### PR TITLE
Updating script to remove dependence on OKTA_DOMAIN env var for test

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
     "test:okta-hosted-login": "protractor okta-oidc-tck/e2e-tests/okta-hosted-login/conf.js",
     "custom-login-server": "npm start --prefix custom-login/",
     "test:custom-login": "protractor okta-oidc-tck/e2e-tests/custom-login/conf.js",
-    "resource-server": "mvn -Dokta.oauth2.issuer=https://${OKTA_DOMAIN}.com/oauth2/default -f samples-java-spring-mvc/resource-server/pom.xml",
-    "test:e2e": "cross-env TEST_TYPE=implicit && npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
```
Comment (vijetmahabaleshwar-okta) [inline package.json]: `cross-env` doesn't export env vars in the context of the new process started by TCK. `export` works fine but needs to be changed for support on windows later.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -52,31 +52,26 @@ In addition to running npm install in the root of the project (to install the de
...
-export CLIENT_ID={{yourAppClientId}}
-export OKTA_DOMAIN={{yourOktaOrgDomain}}
-```
-
-> **NOTE:** Use only the domain part of your org url while setting OKTA_DOMAIN environment.
+* A SPA application
+* A test user account with a known username and password. Note that the USERNAME should be of the form "username@email.com"
 
-* E.g - If your org url is https://myorg.oktapreview.okta.com, your OKTA_DOMAIN should be myorg.oktapreview
+Then you need to setup the following environment variables
```
Comment (jmelberg-okta) [inline CONTRIBUTING.md]: I think we can remove this line - as its purpose is repeated in the following sentence.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -52,31 +52,26 @@ In addition to running npm install in the root of the project (to install the de
...
 
-Then you need to setup the following environment variables
-
-```bash
-export CLIENT_ID={{yourAppClientId}}
-export OKTA_DOMAIN={{yourOktaOrgDomain}}
-```
-
-> **NOTE:** Use only the domain part of your org url while setting OKTA_DOMAIN environment.
+* A SPA application
```
Comment (jmelberg-okta) [inline CONTRIBUTING.md]: In our `README` we link out to "Setting up a SPA" - in this case lets link to our `README` for setup instructions.

File: CONTRIBUTING.md (hunk 2 above)
Comment (vijetmahabaleshwar-okta) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 3 above)
Comment (vijetmahabaleshwar-okta) [reply CONTRIBUTING.md]: Good point. Updated.

**Pull request merged**